portfolio/src/main/webapp/test.js

```diff
@@ -0,0 +1,70 @@
+const one = document.querySelector('#box-area');
```

Comment:
nit: recommend having a comment at the top of this file as well as on top of main.js to say what each are responsible for :)

Reply:
Did one better and just renamed the files properly so that they're descriptive instead of being generic and unhelpful. Also added description comments at the top of each file.

Reply:
Awesome! nicely readable :)